packages/ui-extensions/src/types/index.ts

```diff
@@ -1,3 +1,3 @@
-export * from './Banner';
-export * from './Button';
-export * from './Heading';
+export type {BaseBannerProps} from './Banner';
+export type {BaseButtonProps} from './Button';
+export type {BaseHeadingProps} from './Heading';
```

Comment:
gotta do the same thing here that we had to do in `admin-ui-extensions`: all exported interfaces need to use both `export type` and be named (no `*`). the algorithm we're using to parse imports isn't sophisticated enough to handle `*`.

Reply:
cc @elanalynn

Reply:
i wonder if I can set up a linting rule to disallow `export *`...

Reply:
Sure can:

*eslintrc.js*
```
module.exports = {
  rules: {
    'no-restricted-syntax': [
      {
        selector: 'ExportAllDeclaration',
        message: 'Error message goes here',
      },
    ],
  },
},
```

If you want it to only target certain filepaths you can add it into [overrides](https://eslint.org/docs/user-guide/configuring/configuration-files#how-do-overrides-work) instead

Reply:
![faf](https://user-images.githubusercontent.com/3248903/125668814-75675ec7-4ac6-4e40-9f1c-dc876cd60de7.jpg)